docs/ops/state_backends.md

```diff
@@ -124,7 +124,7 @@ RocksDBStateBackend is currently the only backend that offers incremental checkp
 ## Configuring a State Backend
 
 State backends can be configured per job. In addition, you can define a default state backend to be used when the
-job does not explicitly define a state backend.
+job does not explicitly define a state backend. Besides, state backend configured per-job will overwrite the default state backend configured in `flink-conf.yaml`
```

Comment:
Yeah, this one works too.

How about "State backends can be configured per job in code. In addition, you can define a default state backend in flink-conf.yaml that is used when the job does not explicitly define a state backend."?